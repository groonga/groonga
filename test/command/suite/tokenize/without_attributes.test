#@require-feature mecab

tokenize \
  --tokenizer TokenMecab \
  --string 'Groongaはオープンソースのカラムストア機能付き全文検索エンジンです。' \
  --without_attributes
