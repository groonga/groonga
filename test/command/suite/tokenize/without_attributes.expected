tokenize   --tokenizer TokenMecab   --string 'Groongaはオープンソースのカラムストア機能付き全文検索エンジンです。'   --without_attributes
[
  [
    0,
    0.0,
    0.0
  ],
  [
    "value": "Groonga",
    "value": "は",
    "value": "オープン",
    "value": "ソース",
    "value": "の",
    "value": "カラム",
    "value": "ストア",
    "value": "機能",
    "value": "付き",
    "value": "全文",
    "value": "検索",
    "value": "エンジン",
    "value": "です",
    "value": "。"
  ]
]
